### DISCUSSION TITLE
VIsual studio code issues

### DISCUSSION THREAD

**Discussion by SkanderBahrini** [tailwindlabs/tailwindcss#3735] [Help]
Hello, i got issues while runiing tailwind on VS code as some file are not supported by the IDE.   Thank you

**Comment by NovusOpus** [comment]
Can you be more specific about the problem? I'm guessing you mean Formatting problems?

Do you have **Tailwind CSS IntelliSense**: https://tailwindcss.com/docs/intellisense
